### DISCUSSION TITLE
How to quit all applications before I switch to a bunch?

### DISCUSSION THREAD

**Discussion by yayiji** [ttscoff/bunch-docs#227] [Q&A] [answered]
Hi, everyone! Is there a way to quit all applications in Bunch when I switch to a particular bunch?

**Comment by ttscoff** [comment]
Yes, but I need a better explanation of what you mean. Do you have a bunch open and want to close it automatically when you open another bunch? Do you want to close a particular bunch when another justiciar bunch opens? Are you airways closing the same bunch? Are you asking if you can only quit aid without performing any other on-close actions?

-Brett
________________________________
From: Yiji Ya ***@***.***>
Sent: Monday, July 11, 2022 1:54:06 AM
To: ttscoff/bunch ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ttscoff/bunch] How to quit all applications before I switch to a bunch? (Discussion #227)


Hi, everyone! Is there a way to quit all applications in Bunch when I switch to a particular bunch?

**Reply by yayiji** [reply]
I mean, quit all apps I have opened on Mac, then execute a bunch to open a new workspace just like when I restart the computer and then run a start bunch.

**Comment by ttscoff** [comment, marked as answer]
https://bunchapp.co/docs/bunch-files/commands/quit-everything/

**Reply by yayiji** [reply]
Thank you

**Comment by ttscoff** [comment]
Use:

(quit everything)


At the beginning of the Bunch.

-Brett
